lib/sycamore/sycamore/transforms/llm_query.py

```diff
@@ -22,16 +22,17 @@ class LLMTextQueryAgent:
         llm_kwargs: (Optional) LLM keyword argument for the underlying execution engine
         per_element: (Optional) Whether to execute the call per each element or on the Document itself. Defaults to
         True.
+        element_type: (Optional) Element type based filter to make call for only specific element type. Defaults to Any
```

Comment:
I think we need to be more clear on what this means. It's a string. What values can it take? Should we use a python `typing.Type` for this instead?